ios/MullvadVPN/AppDelegate.swift

```diff
@@ -21,21 +20,19 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     let simulatorTunnelProvider = SimulatorTunnelProviderHost()
     #endif
 
-    private var loadTunnelSubscriber: AnyCancellable?
-    private var refreshTunnelSubscriber: AnyCancellable?
-
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         #if targetEnvironment(simulator)
         SimulatorTunnelProvider.shared.delegate = simulatorTunnelProvider
         #endif
 
         let accountToken = Account.shared.token
 
-        loadTunnelSubscriber = TunnelManager.shared.loadTunnel(accountToken: accountToken)
-            .receive(on: DispatchQueue.main)
-            .sink(receiveCompletion: { (completion) in
-                if case .failure(let error) = completion {
-                    fatalError("Failed to restore the account: \(error.localizedDescription)")
+        RelayCache.shared.updateRelays()
```

Comment:
Update relays each time the GUI app starts in case if the tunnel doesn't run and can't provide updates on time. Otherwise the tunnel runs periodic updates.